lib/ansible/constants.py

```diff
@@ -139,41 +139,46 @@ def __getitem__(self, y):
 INTERNAL_RESULT_KEYS = ('add_host', 'add_group')
 INTERNAL_STATIC_VARS = frozenset(
     [
-        "ansible_async_path",
+        "ansible_check_mode",
         "ansible_collection_name",
         "ansible_config_file",
```

Comment:
I missed a few more...
```suggestion
        "ansible_config_file",
        "ansible_current_hosts",
        "ansible_delegated_vars",
```